src/layout.jl

```diff
@@ -259,7 +261,7 @@ function _spectral(A::SparseMatrixCSC)
     D = sparse([1:length(data)], [1:length(data)], data)
     L = D - A
     ncv = max(7, int(sqrt(length(data))))
-    eigenvalues, eigenvectors = eigs(L, nev=3, ncv=ncv)
+    eigenvalues, eigenvectors = LightGraphs.LinAlg.eigs(L, nev=3, ncv=ncv)
```

Comment:
you will need to specify the which=LR() for this eigs call.
and using ArnoldiMethods in order for this to work

Reply:
Ah I see, I though I thought `LightGraphs.LinAlg.eigs`  was just an identical replacement for the old `eigs` functions. Any reason why I should use `LightGraphs.LinAlg.eigs` instead of `partialschur` directly?